### DISCUSSION TITLE
[Read this] If your bot is stuck in an application...

### DISCUSSION THREAD

**Discussion by voidbydefault** [voidbydefault/EasyApplyBot#26] [Q&A]
Please help me improve the bot's error handling by submitting `text` and an image of the step of the application where it was stuck. Examples of errors that push the bot into an endless loop include:

```
English:

- please enter a valid answer
- file is required
- please make a selection
```

```
Non-English:

- saisissez un numéro
etc.
```

Please submit similar errors in this thread, whether in English or otherwise. Thanks.

**Comment by syosry** [comment]
![Screen Shot 2024-07-08 at 11 06 29 PM](https://github.com/voidbydefault/EasyApplyBot/assets/175078880/0b85ff31-15cc-428a-a95e-cf009b7d6eb3)

**Comment by syosry** [comment]
Hi I hope you are doing well I am running the most recent Sep 24 code and I am getting this error if you could please assist. Thanks!
![Screen Shot 2024-10-19 at 12 45 22 AM](https://github.com/user-attachments/assets/ed970eee-7efe-404d-af28-ab483e9460c6)

**Reply by voidbydefault** [reply]
It's because you have not configured your profile in `config.yaml`. I have added detailed explanation in it, follow it properly before running the bot.

**Comment by syosry** [comment]
Hi,

Thank you for your reply. I’m a bit confused I had downloaded the September
2024 update and filled out all my information in the new config.yaml. Where
can I find the detailed explanation?

**Comment by voidbydefault** [comment]
What kind of explanation?

`config.yaml` has comments to explain every choice available:
![image](https://github.com/user-attachments/assets/b1939df5-ec8d-496c-b699-4f69e5caaceb)

Watch this video which talks about Python setup, etc.
https://www.youtube.com/watch?v=oQEAcAur4Vk
